app/models/setting.rb

```diff
@@ -71,7 +72,15 @@ def self.config_file
   end
 
   def self.live_descendants
-    self.disabled_plugins.default_organization.organization_fact.default_location.location_fact
+    self.disabled_plugins.default_organization.organization_fact.default_location.location_fact.order_by(:category)
+  end
+
+  def self.stick_general_to_first
+    where(:category => sticky_setting).concat(where.not(:category => sticky_setting)).group_by(&:category)
+  end
+
+  def self.sticky_setting
```

Comment:
I am not a fan of private class methods. Moved it to a variable in `stick_general_first`.